app/models/concerns/account/header.rb

```diff
@@ -4,7 +4,7 @@ module Account::Header
   extend ActiveSupport::Concern
 
   IMAGE_MIME_TYPES = ['image/jpeg', 'image/png', 'image/gif', 'image/webp'].freeze
-  LIMIT = 2.megabytes
+  LIMIT = 10.megabytes
```

Comment:
It might make sense to use the same logic as for the Preview Cards:
```suggestion
  LIMIT = Rails.configuration.x.use_vips ? 8.megabytes : 2.megabytes
```

Reply:
Is it likely that for the foreseeable future the combination of mime types and the file size limits on preview cards, avatars, headers - will be in sync w/ each other?

Reply:
Not so much, I think, as they have pretty different purposes. What we're discussing here is a stopgap measure, but we probably want client-side processing and better error states.